pom.xml

```diff
@@ -13,7 +13,7 @@
   <inceptionYear>2004</inceptionYear>
 
   <prerequisites>
-    <maven>2.2.1</maven>
+    <maven>3.2.2</maven>
```

Comment:
@stephenc yes +1000
@MarkEWaite the choice of 3.2.2 is coming from the report done by `mvn versions:display-plugin-updates` which is verifying the requirement of each plugin declared in the POM. But yes we may target the latest 3.2.x